src/CommonParametersTrait.php

```diff
@@ -328,4 +328,14 @@ public function setNewCustomerReferenceCode($value)
     {
         return $this->setParameter('newCustomerReferenceCode', $value);
     }
+
+    public function getAvsResponse()
+    {
+        return $this->getParameter('AVSResponse');
+    }
+
+    public function setAvsResponse($value)
+    {
+        return $this->setParameter('AVSResponse', $value);
+    }
```

Comment:
I don't think these are necessary. Aren't these just getters/setters for sending the request?